artifacts/argocd-apps/grafana-operator-chart/templates/amg_grafana-amp-datasource.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: grafanadatasource-amp
+  namespace: grafana-operator
+spec:
```

Comment:
These file - [https://github.com/aws-observability/aws-observability-accelerator/pull/28/files#diff-f0c4fb63114ba78a5db50b55191[…]d8d51abe9b7ff3a5abcc9cd2d375a2f8](https://github.com/aws-observability/aws-observability-accelerator/pull/28/files#diff-f0c4fb63114ba78a5db50b5519126758d8d51abe9b7ff3a5abcc9cd2d375a2f8) Can we not use the original files in GitOps repo which we use for flux

like this [https://github.com/aws-observability/aws-observability-accelerator/blob/main/artif[…]or-manifests/eks/infrastructure/amg_grafana-amp-datasource.yaml](https://github.com/aws-observability/aws-observability-accelerator/blob/main/artifacts/grafana-operator-manifests/eks/infrastructure/amg_grafana-amp-datasource.yaml)

Reply:
Difference b/w existing one and new one is, new one is compatible for ArgoCD based on values.. whereas old one is compatible with FluxCD kustomization.